ext/express-parser/express_parser.cpp

```diff
@@ -11,6 +11,15 @@
 #include "rice/Constructor.hpp"
 #include "rice/Director.hpp"
 
+#ifdef _WIN32
+// windef.h we will resists you!
+#undef FALSE
+#undef TRUE
+#undef OPTIONAL
+#undef IN
+#undef OUT
+#endif
+
```

Comment:
This file is autogenerated with https://github.com/camertron/antlr4-native-rb. I make a few changes in it during the generation process in https://github.com/lutaml/expressir/blob/cpp-parser/exe/generate-parser#L23-L43, but I'd want to to send a PR upstream with them. I'm not sure if these Windows-specific lines shall be in the PR. Could they be moved to a separate file?

Reply:
If those files autogenerated I think we should move them to the grammar file or to the script mentioned above. I will check this